docs/Guide/plugins/i18next/getting-started.mdx

```diff
@@ -49,6 +49,57 @@ import '@sapphire/plugin-i18next/register';
 </TabItem>
 </Tabs>
 
+## Get the language set per server
+
+To obtain the language that each server has configured, as well as the custom prefixes for each server, for that we will
+have now an option called `i18n` in the Sapphire client options, in which we will do the following.
+
+```typescript ts2esm2cjs|{3}|{3}
+import { SapphireClient } from '@sapphire/framework';
+import { InternationalizationContext } from '@sapphire/plugin-i18next';
+import '@sapphire/plugin-i18next/register';
```

Comment:
```suggestion
```

register breaks the convertion so it needs its own codeblock which is already above.